web/src/Components/Solution/SolutionData.tsx

```diff
@@ -34,6 +35,11 @@ interface SacksProps {
   totalMass: number
 }
 
+interface DensitiesProps {
+  products: Products
+  productResults: Array<ProductResult>
+}
+
 const Sacks = ({ products, productResults, totalMass }: SacksProps): ReactElement => {
```

Comment:
All things "Sacks" in SolutionData is never used right? Same with the "usingSacks" bool.
If so, we should remove dead code.